src/widgetastic_patternfly5/components/date_and_time/calendar_month.py

```diff
@@ -18,17 +17,15 @@ class BaseCalendarMonth:
     """
 
     CALENDAR_HEADER = ".//div[contains(@class, '-c-calendar-month__header')]"
-    MONTH_SELECT_LOCATOR = f"{CALENDAR_HEADER}//div[contains(@class, 'header-month')]"
+    MONTH_SELECT_LOCATOR = ".//div[contains(@class, '-c-calendar-month__header-month')]"
```

Comment:
Why not change CALENDAR_HEADER?  Looks like all uses of it are removed, and the difference is including `-month` suffix on the class name.